### PR TITLE
dotCMS/core#25092 fix comment - trim spaces

### PR DESCRIPTION
**Pull request by oidacra**
### Proposed Changes
* Trim spaces of a valid input value

### CODE REVIEW

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-inline-edit-text/dot-experiments-inline-edit-text.component.ts [hunk 1]
```diff
@@ -150,7 +156,7 @@ export class DotExperimentsInlineEditTextComponent implements OnChanges {
      */
     saveAction(): void {
         if (this.textControl.valid) {
-            this.textChanged.emit(this.textControl.value);
+            this.textChanged.emit(this.textControl.value.trim());
         }
     }
```
Comment (zJaaal) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-inline-edit-text/dot-experiments-inline-edit-text.component.ts:161]: can we test this?

**Review by zJaaal — DISMISSED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-inline-edit-text/dot-experiments-inline-edit-text.component.ts (hunk 1 above)
Comment (oidacra) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/shared/ui/dot-experiments-inline-edit-text/dot-experiments-inline-edit-text.component.ts:161]: Added,

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html [hunk 2]
```diff
@@ -126,6 +126,7 @@ <h2 class="flex align-items-center gap-1 uppercase">
                     [required]="true"
                     [text]="variant.name"
                     (textChanged)="editVariantName($event, variant, vm.experimentId)"
+                    hideErrorMsg="true"
```
Comment (fmontes) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html:129]: test?

Comment (fmontes) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html:129]: Test?

**Review by fmontes — DISMISSED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html (hunk 2 above)
Comment (rjvelazco) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html:129]: Shouldn't it be?

```suggestion
                    [hideErrorMsg]="true"
```

**Review by rjvelazco — DISMISSED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html (hunk 2 above)
Comment (oidacra) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-variants/dot-experiments-configuration-variants.component.html]: - Added test.
-  Changed input name and logic, removed.

**Review by zJaaal — APPROVED** [review]

**Review by fmontes — APPROVED** [review]

**Review by hmoreras — APPROVED** [review]

**Pull request merged**